### DISCUSSION TITLE
Opinion: The name is too ambigious for what this extension does and the value it brings.

### DISCUSSION THREAD

**Discussion by abreinig** [mbnuqw/sidebery#766] [Ideas]
To Mbnuqw and all the contributors... all of you are AWESOME. This is by far the best tree tabs extension and I regret that I didn't find it sooner ( Coming from "Tree Tabs") .

Speaking of finding it sooner, I think if you incorporated "tree tabs" or something like that into the name of this extension it would help people like me find it a lot faster. Perhaps "Sidebery - Tree Style Tabs". I know I personally would have found it much sooner if it was named differently.

Thank you again for making and maintaining this extension!

**Comment by Qjo1** [comment]
Interesting thought. In the same spirit, including 'vertical' might help too, as there's many who seek to mimic Edge's vertical tabs (although Sidebery is arguably even better). That's actually how I came to try it out but I don't remember how I found out about it.

So maybe: Sidebery - Vertical Tabs and Tab Trees?

**Comment by davidleejy** [comment]
> [...] extension it would help people like me find it a lot faster. [...]

Just an idea: if it's a search problem, can the relevant phrases suggested by well-meaning users like "vertical tab", "tab tree", etc. be incorporated into non-title fields commonly thought to undergo indexing by search mechanisms? how about the description field or the tags field?

**Comment by michalfita** [comment]
These days it's pretty easy to find Sideberry if you're looking for Tree Style Tabs (it was killing my PC) alternatives.
